ftw/linkchecker/linkchecker.py

```diff
@@ -0,0 +1,100 @@
+from multiprocessing import Pool
+import time
+import requests
+import urls_testing
+import os
+import sys
+
+urls = urls_testing.urls
```

Comment:
I'd move that to `__main__`, since it is only for testing.

Reply:
I moved it to `work_through_urls` where I'm actually using it.